config.ru

```diff
@@ -1,3 +1,15 @@
 require './config/environment'
+require 'plezi'
 
+class Controller
+  def index
+    'Hello from Plezi!'
+  end
+end
+
+Plezi.route '/plezi', Controller
+
+app = proc { |_env| [200, { 'Content-Length' => '11' }, ['Hello Rack!']] }
```

Comment:
Is this proc used anywhere? I might be misunderstanding something.

Reply:
I don't know, I'm not sure what it's for to be entirely honest, it was just in the [example](http://www.plezi.io/docs/with_rack_app).

I just checked and it runs without it since yeah, we aren't doing anything with it.